spec/services/html/image_uri_spec.rb

```diff
@@ -0,0 +1,36 @@
+require "rails_helper"
+
+RSpec.describe Html::ImageUri, type: :service do
+  let(:camo) { "https://camo.githubusercontent.com/64b9f7c7c5f41ec22113b61235256435cd61779a0554b0595b88b6011f94c60b/68747470733a2f2f696d672e736869656c64732e696f2f6769746875622f636f6d6d69742d61637469766974792f772f666f72656d2f666f72656d" }
+  let(:badge) { "https://github.com/forem/forem/actions/workflows/ci-cd.yml/badge.svg?branch=depfu/update/sterile-1.0.24" }
```

Comment:
Just want to mention: I don't really know how brittle this set up is going to be in the long-term. Basically, there's very little guarantee that GitHub actions will continue to use `badge.svg` _forever_. I'm not aware of a better or more obvious indicator that any particular `svg` is _legitimately_ coming from workflow actions.

**Also**, detecting on the filename potentially allows someone to "inject" an svg if they give it this exact name. On the other hand, we're allowing _everything_ from `camo.githubusercontent.com` which is effectively _anything_ anyone drags into a comment area, so I don't think we're at an increased risk from this change.